src/pages/liked-videos/LikedVideos.jsx

```diff
@@ -1,12 +1,11 @@
-import "./liked-videos.css";
 import { VideoCard } from "../../components";
 import {useVideos} from "../../context/video-context";
```

Comment:
Suggestion: use absolute path instead of the relative path as it increases the code readability.

Reply:
Thank you for the suggestion. Will try to implement it later.